budgetApp1/budgetApp1/UserViewModel.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class UserViewModel {
+    
+    // MARK: - Singleton instantiation of ViewModel
+    static let shared = UserViewModel()
```

Comment:
I am using a singleton because for some reason when I call the user stored in the userVM in the Login screen, it is empty. I think it is because when I instantiate the userVM again in the login after the user has signed in, it loses the value stored in User so I thought using a singleton would make the details persistent across both Login and SignUp Controllers.
I know it's bad coding principles but I just had to find a way around this.

Reply:
had a chat with you on a call. We will use coredata in the next PR